zh_cn.lang

```diff
@@ -355,58 +355,58 @@ effect.xaerominimap.no_minimap_beneficial=小地图不可用
 effect.xaerominimap.no_entity_radar_beneficial=实体雷达不可用
 effect.xaerominimap.no_waypoints_beneficial=路径点不可用
 effect.xaerominimap.no_cave_maps_beneficial=洞穴地图不可用
-gui.xaero_radar_item_frames=用圆点显示物品展示框
-gui.xaero_show_dimension_name=显示当前位面
+gui.xaero_radar_item_frames=用点显示物品展示框
+gui.xaero_show_dimension_name=显示当前维度
 gui.xaero_unknown_biome=未知生物群系
-gui.xaero_light_value=光照强度
-gui.xaero_teleport_y_unknown=你无法传送到一个高度未知的路径点，请编辑路径点信息以确保能成功传送。
-gui.xaero_box_waypoints_distance_scale2=游戏内路径点距离显示文本的尺寸。
-gui.xaero_box_waypoints_name_scale2=游戏内路径点名称显示文本的尺寸。
-gui.xaero_box_ingame_waypoint_icon_scale=游戏内路径点图标的尺寸。
-gui.xaero_ingame_waypoint_icon_scale=游戏内路径点图标尺寸
-gui.xaero_ingame_waypoint_close_scale=游戏内路径点接近时尺寸
-gui.xaero_box_ingame_waypoint_close_scale=当游戏内路径点近在咫尺时显示的大小。如要改变远处路径点的大小，请更改“游戏内路径点图标尺寸”，“路径点距离文本尺寸”和“路径点名称文本尺寸”设置。
-gui.xaero_entity_icon_name_fallback=加载图标失败时显示实体名称
+gui.xaero_light_value=光照等级：%d
+gui.xaero_teleport_y_unknown=你不能传送到未指定 Y 坐标的路径点。编辑路径点以便能够传送到它的位置。
+gui.xaero_box_waypoints_distance_scale2=到游戏内路径点距离文本显示缩放。
+gui.xaero_box_waypoints_name_scale2=游戏内路径点名称文本显示缩放。
+gui.xaero_box_ingame_waypoint_icon_scale=游戏内路径点图标的缩放。
+gui.xaero_ingame_waypoint_icon_scale=游戏内路径点图标缩放
+gui.xaero_ingame_waypoint_close_scale=近距离游戏内路径点缩放
+gui.xaero_box_ingame_waypoint_close_scale=当你非常靠近它们时，游戏内路径点的缩放。 要更改远处路径点的缩放，请使用 “游戏内路径点图标缩放”、“路径点距离文本缩放”和“路径点名称文本缩放”设置。
+gui.xaero_entity_icon_name_fallback=找不到图标时使用实体名称
 gui.xaero_settings_search=设置搜索
 gui.xaero_settings_search_placeholder=搜索……
-gui.xaero_minimap_view_settings=显示设置
+gui.xaero_minimap_view_settings=视图设置
 gui.xaero_entity_radar_settings=实体雷达设置
 gui.xaero_entity_radar_categories=实体类别设置
 gui.xaero_minimap_block_map_settings=方块地图设置
-gui.xaero_overlay_settings=Overlay Settings
+gui.xaero_overlay_settings=叠加层设置
 gui.xaero_minimap_info_settings=信息设置
 gui.xaero_waypoint_settings=路径点设置
-gui.xaero_minimap_misc_settings=杂项设置
+gui.xaero_minimap_misc_settings=其他设置
 gui.xaero_entity_radar_category=类别：
-gui.xaero_settings_not_found=找不到与您的输入匹配的项！
+gui.xaero_settings_not_found=找不到与你的输入匹配的项！
 gui.xaero_multiple_waypoint_info=多个路径点距离
-gui.xaero_box_multiple_waypoint_info=多个游戏内路径点的距离是否一次性显示。默认情况下，只有距离准星最近的路径点的距离会显示。使用“路径点距离显示”设置以配置哪些路径点的距离会显示。
+gui.xaero_box_multiple_waypoint_info=是否可以同时显示多个游戏内路径点的距离。 默认情况下，仅显示离十字准星最近的路径点的距离。 使用“到路径点的距离”设置来配置可以显示到哪些路径点的距离。
```

Comment:
The old translation feels more correct. Please make sure it works for
`Whether the distance can be displayed for multiple in-game waypoints at once. By default, only the distance to the closest waypoint to the crosshair is displayed. Use the "Distance to WP" setting to configure which waypoints the distance could even be shown for.`